android/app/src/main/java/org/fossasia/openevent/dbutils/DatabaseOperations.java

```diff
@@ -161,6 +161,29 @@ public Session getSessionById(int id, SQLiteDatabase mDb) {
         return location;
     }
 
+    public int getSpeakerIdbySessionId(int id, SQLiteDatabase mDb) {
+        String selection = DbContract.Sessionsspeakers.SESSION_ID + EQUAL + id;
+        Cursor cur = mDb.query(
+                DbContract.Sessionsspeakers.TABLE_NAME,
+                DbContract.Sessionsspeakers.FULL_PROJECTION,
+                selection,
+                null,
+                null,
+                null,
+                null
+        );
+
+        int speakerId = 0;
+        cur.moveToFirst();
+        while (!cur.isAfterLast()) {
+            speakerId = cur.getInt(cur.getColumnIndex(DbContract.Sessionsspeakers.SPEAKER_ID));
```

Comment:
@the-dagger There can be multiple speakers in one session. How do you handle that?